### PR TITLE
Add test for unary minus operator with float

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/conformance/glsl/bugs/00_test_list.txt [hunk 1]
```diff
@@ -35,5 +35,6 @@
 --min-version 1.0.4 sketchfab-lighting-shader-crash.html
 --min-version 1.0.4 struct-constructor-highp-bug.html
 --min-version 1.0.3 temp-expressions-should-not-crash.html
+unary-minus-operator-float-bug.html
```
Comment (zhenyao) [inline sdk/tests/conformance/glsl/bugs/00_test_list.txt]: Please add min-version 1.0.4

File: sdk/tests/conformance/glsl/bugs/unary-minus-operator-float-bug.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+function runTest() {
+    var width = 64;
+    var height = 64;
+
+    var program = wtu.setupProgram(gl, ["vshader", "fshader"], ['a_position']);
+    gl.viewport(0, 0, width, height);
+    wtu.setupUnitQuad(gl, 0, 1);
+    wtu.drawUnitQuad(gl);
+    var msg = "";
+    var tolerance = 7;
```
Comment (zhenyao) [inline sdk/tests/conformance/glsl/bugs/unary-minus-operator-float-bug.html]: I think 3 is already very allowing. 7 is too big.

File: sdk/tests/conformance/glsl/bugs/00_test_list.txt (hunk 1 above)
Comment (qkmiao) [inline sdk/tests/conformance/glsl/bugs/00_test_list.txt]: Done.

File: sdk/tests/conformance/glsl/bugs/unary-minus-operator-float-bug.html (hunk 2 above)
Comment (qkmiao) [inline sdk/tests/conformance/glsl/bugs/unary-minus-operator-float-bug.html]: Set tolerance as 3.

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**